hometask5

```diff
@@ -0,0 +1,55 @@
+#Task 1. Fibonacci generator
+
+a = int(input('Input a number: '))
+
+
+def my_generator(n):
+    a=b=1
+    for item in range(n):
+        yield a
+        a, b = b, a+b
+
+print(list(my_generator(a))[-1])
+
+
+
+#Task 2. Fibonacci iterator
+
+a = int(input('Input a number: '))
+
+fibonacci_numbers = [0, 1]
+for i in range(2,700): #fibonacci numbers up to 700, don't know how to make to infinity
+    fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
+def my_iterator(n):
+    a=b=1
+    for item in range(n):
+        yield a
```

Comment:
This is not an iterator, this is a generator

Reply:
Made an update